pkg/controllers/controller.go

```diff
@@ -17,7 +17,6 @@ package controllers
 import (
 	"context"
 	"fmt"
-
```

Comment:
I think your editor's style is fighting VScode here.

Reply:
I'm using `make ci` to do this, not using my IDEs formatter.